core/storage/blog/gae_models_test.py

```diff
@@ -492,7 +492,9 @@ def test_deassign_user_from_all_blog_posts(self) -> None:
             self.assertTrue(self.USER_ID_NEW not in model.editor_ids)
 
     def test_deassign_user_from_blog_post_handles_invalid_user_id(self) -> None:
-        # Raises no error.
+        # If the BlogPostRightsModel has not been created for a User, the
```

Comment:
Each BlogPostRightsModel is for only one blog post.

So when you talk about "the BlogPostRightsModel for a user", what does that mean?

Reply:
Updated the comment now. If the user is not in the editor list of any blog post, the method `BlogPostRightsModel.deassign_user_from_all_blog_posts()` should do nothing and exit.